tests/v2/validation/rbac/snapshotrbac/snapshot_rbac_test.go

```diff
@@ -59,10 +59,22 @@ func (etcd *SnapshotRBACTestSuite) testRKE2K3SSnapshotRBAC(role string, standard
 	}
 }
 
-func (etcd *SnapshotRBACTestSuite) TestRKE2K3SSnapshotRBAC() {
-	subSession := etcd.session.NewSession()
-	defer subSession.Cleanup()
+func (etcd *SnapshotRBACTestSuite) testRKE1SnapshotRBAC(role string, standardUserClient *rancher.Client) {
+	log.Info("Test case - Take Etcd snapshot of an RKE1 cluster as a " + role)
+	err := etcdsnapshot.CreateRKE1Snapshot(standardUserClient, etcd.cluster.Name)
+	switch role {
+	case rbac.ClusterOwner.String(), rbac.RestrictedAdmin.String():
+		require.NoError(etcd.T(), err)
+		log.Info("Snapshot successful!!!")
+
+	case rbac.ClusterMember.String(), rbac.ProjectOwner.String(), rbac.ProjectMember.String():
+		require.Error(etcd.T(), err)
+		assert.Contains(etcd.T(), err.Error(), "action [backupEtcd] not available")
+		log.Info("Snapshot successful!!!")
```

Comment:
Shouldn't this log message reflect that the snapshot failed as expected? Perhaps something like: `log.Info("Snapshot failed as expected")`

Reply:
Yes that's a good point. I will update it accordingly.